### ISSUE TITLE
Root as Guest

### ISSUE THREAD

**Issue by CjMoore** [CjMoore/cloney_island#1]
As a Guest
	When I visit ‘/’ 
	I see a list of all projects
	I should see a button to log in

**Comment by CjMoore** [comment]
- "projects#index"

**Comment by CjMoore** [comment]
- create home controller
- create projects model and migration

**Comment by s-espinosa** [comment]
@CjMoore Believe this is now a subset of all projects? Think that's fine, but wanted to confirm before I moved to done.

**Comment by CjMoore** [comment]
Correct. I believe we decided to go with displaying projects that are very close to being funded on the home page.

**Comment by CjMoore** [comment]
That was an accident.
